banking/src/main/java/rocks/kata/DatabaseClasses/AccountOperationsDatabase.java

```diff
@@ -0,0 +1,37 @@
+package rocks.kata.DatabaseClasses;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import rocks.kata.AccountOperation;
+
+public class AccountOperationsDatabase {
+    private Map<Long, List<AccountOperation>> operationsDatabase;
+    private AccountOperationDatabaseFormatter formatter;
+
+    public AccountOperationsDatabase(AccountOperationDatabaseFormatter formatter) {
+        this.operationsDatabase = new HashMap<>();
+        this.formatter = formatter;
+    }
+
+    public void addOperationToDatabase(long accountID, AccountOperation operation) {
+        if (!operationsDatabase.keySet().contains(accountID)) {
```

Comment:
`Map` ma kilka ciekawych metod ;)

https://nurkiewicz.com/2019/03/mapmerge-one-method-to-rule-them-all.html

Reply:
Nie udało mi się znaleźć ładnego sposobu na metodę merge, więc chętnie przyjmę wskazówki :D Still putIfAbsent dobra rzecz